worlds/_bizhawk/context.py

```diff
@@ -236,9 +236,9 @@ def _patch_and_run_game(patch_file: str):
         metadata, output_file = Patch.create_rom_file(patch_file)
         Utils.async_start(_run_game(output_file))
         return metadata
-    except Exception as exc:
-        logger.exception(exc)
-        return {}
+    except Exception as e:
+        Utils.messagebox('Error', str(e), True)
+        raise
```

Comment:
```suggestion
    except Exception as exc:
        logger.exception(exc)
        Utils.messagebox("Error Patching Game", str(e), True)
        return {}
```
No need to remove the stuff we already have.

Reply:
At the time I assumed it was a replacement for what was there, since SNI client (which I was referred to) only had the messagebox. Lesson of today is to not open PRs when tired

Reply:
Whoops, that syntax error is my bad. Forgot to change `str(e)` to `str(exc)`